pkg/validate/pod.go

```diff
@@ -32,7 +33,46 @@ var _ = framework.KubeDescribe("Test CRI PodSandbox", func() {
 		c = f.CRIClient.CRIRuntimeClient
 	})
 
-	It("test get version info", func() {
+	It("test get version info [Conformance]", func() {
```

Comment:
We should make this a complete sentence (and all followings), e.g. `runtime should return version info`. And `Version()` should be outside of sandbox context.